packages/calcite-components/src/components/table-cell/table-cell.scss

```diff
@@ -128,22 +136,15 @@ td.last-cell {
 }
 
 .selection-cell.selected-cell {
-  box-shadow: var(--calcite-table-cell-shadow-selected, inset 0.25rem 0 0 0 var(--calcite-color-brand));
-  color: var(--calcite-table-cell-text-color-selected, var(--calcite-color-brand));
-  & calcite-icon {
-    --calcite-icon-color: var(--calcite-table-cell-text-color-selected, var(--calcite-color-brand));
+  box-shadow: inset 0.25rem 0 0 0 var(--calcite-table-cell-indicator-color-selected, var(--calcite-color-brand));
```

Comment:
This doesn't address my issue with this naming change - this is really a row-specific visual display and I think we should keep that nomenclature. I still think it should use table-row terminology, and that "--calcite-table-row-selected-accent-color" is a more apt name. "selected-highlight, "selected-indicator") all work for me, as long as its scoped to the row.